pkg/api/dtos/models.go

```diff
@@ -61,7 +62,9 @@ func GetGravatarUrl(text string) string {
 	}
 
 	hasher := md5.New()
-	hasher.Write([]byte(strings.ToLower(text)))
+	if _, err := hasher.Write([]byte(strings.ToLower(text))); err != nil {
+		log.Warn("Failed to hash text: %s", err)
```

Comment:
Need to be `Warnf` to work. But use argument `log.Warn("...", "err", err` instead.

Reply:
@marefr See my above comment.